kyuubi-common/src/main/scala/org/apache/kyuubi/service/authentication/AuthenticationProviderFactory.scala

```diff
@@ -62,4 +64,30 @@ object AuthenticationProviderFactory {
       new AnonymousAuthenticationProviderImpl
     }
   }
+
+  def getHttpBasicAuthenticationProvider(
+      method: AuthMethod,
+      conf: KyuubiConf): PasswdAuthenticationProvider = method match {
+    case AuthMethods.NONE => new AnonymousAuthenticationProviderImpl
+    case AuthMethods.LDAP => new LdapAuthenticationProviderImpl(conf)
+    case AuthMethods.JDBC => new JdbcAuthenticationProviderImpl(conf)
+    case AuthMethods.CUSTOM =>
+      val className = conf.get(KyuubiConf.AUTHENTICATION_CUSTOM_BASIC_CLASS)
+      if (className.isEmpty) {
+        throw new AuthenticationException(
+          "authentication.custom.basic.class must be set for http basic authentication.")
+      }
```

Comment:
this is caused by admin's wrong configuration, should not throw `AuthenticationException`
```
require(className.nonEmpty, "xxxxx")
```

please correct the mentioned configuration key
`authentication.custom.basic.class` => `kyuubi.authentication.custom.basic.class`